app/src/main/java/io/appium/uiautomator2/server/ServerInstrumentation.java

```diff
@@ -78,24 +78,30 @@ private void setAccessibilityServiceState() {
 
     private ServerInstrumentation(Context context, int serverPort, int mjpegServerPort) {
         if (!isValidPort(serverPort)) {
```

Comment:
positive conditions are easier to read than negative. Consider flipping it

Reply:
same below

Reply:
Sure.

Reply:
This will be resolved w/ the upcoming update(s)